tests/upgrade/dashboards_test.js

```diff
@@ -0,0 +1,174 @@
+Feature('PMM upgrade tests for dashboards');
+
+Before(async ({ I }) => {
+  await I.Authorize();
+});
+
+Scenario(
+  'PMM-T391 PMM-T1818 Verify user is able to create and set custom home dashboard @pre-dashboards-upgrade',
+  async ({
+    I, grafanaAPI, dashboardPage, searchDashboardsModal,
+  }) => {
+    const insightFolder = await grafanaAPI.lookupFolderByName(searchDashboardsModal.folders.insight.name);
+
+    await grafanaAPI.createCustomDashboard(grafanaAPI.randomDashboardName, insightFolder.id, null, ['pmm-qa', grafanaAPI.randomTag]);
+    const folder = await grafanaAPI.createFolder(grafanaAPI.customFolderName);
+    let additionalPanel = null;
+
+    const libResp = await grafanaAPI.savePanelToLibrary('Lib Panel', folder.id);
+    const libPanel = libResp.result.model;
+
+    libPanel.libraryPanel.meta = libResp.result.meta;
+    libPanel.libraryPanel.version = 1;
+    libPanel.libraryPanel.uid = libResp.result.uid;
+    additionalPanel = [libPanel];
+
+    const resp = await grafanaAPI.createCustomDashboard(grafanaAPI.customDashboardName, folder.id, null, []);
+
+    await grafanaAPI.starDashboard(resp.id);
+    await grafanaAPI.setHomeDashboard(resp.id);
+
+    I.amOnPage('');
+    dashboardPage.waitForDashboardOpened();
+    // dashboardPage.verifyMetricsExistence(['Custom Panel']);
```

Comment:
Is there a reason why this is commented out?

Reply:
uncommented